lib/killbill_client/models/subscription.rb

```diff
@@ -220,6 +221,23 @@ def undo_change_plan(user = nil, reason = nil, comment = nil, options = {})
                        }.merge(options)
       end
 
+      #
+      # Update Subscription Quantity
+      #
+      def update_quantity(quantity, effective_from_date = nil, force_new_quantity_with_past_effective_date = false, user = nil, reason = nil, comment = nil, options = {})
```

Comment:
I would suggest to keep the parameter ordering similar to the [update_bcd](https://github.com/tungleduyxyz/killbill-client-ruby/blob/c1e238b888984e3c1786a52f4ee86c838aa41db6/lib/killbill_client/models/subscription.rb#L144) method.